<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="inputDirectoryLabel">
        <property name="text">
         <string>&amp;Input directory</string>
        </property>
        <property name="buddy">
         <cstring>inputDirectoryBrowse</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="inputDirectoryLayout">
        <item>
         <widget class="QLineEdit" name="inputDirectoryLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="inputDirectoryBrowse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="outputDirectoryLabel">
        <property name="text">
         <string>&amp;Output directory</string>
        </property>
        <property name="buddy">
         <cstring>outputDirectoryBrowse</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="outputDirectoryLayout">
        <item>
         <widget class="QLineEdit" name="outputDirectoryLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="outputDirectoryBrowse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="processingToolLabel">
        <property name="text">
         <string>Processing &amp;tool</string>
        </property>
        <property name="buddy">
         <cstring>processingToolBrowse</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="processingToolLayout">
        <item>
         <widget class="QLineEdit" name="processingToolLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="processingToolBrowse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="processingToolArgumentsLabel">
        <property name="text">
         <string>Processing tool &amp;arguments</string>
        </property>
        <property name="buddy">
         <cstring>processingToolArgumentsLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="processingToolArgumentsLineEdit">
        <property name="toolTip">
         <string>Use placeholders: {input} for input file path, {output} for output file path</string>
        </property>
        <property name="placeholderText">
         <string>{input} {output}</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fileFilterLabel">
        <property name="text">
         <string>File &amp;filter</string>
        </property>
        <property name="buddy">
         <cstring>fileFilterLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="fileFilterLineEdit">
        <property name="placeholderText">
         <string>*.*</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startStopButton">
        <property name="text">
         <string>&amp;Start</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
